### ISSUE TITLE
object '.pbd_env' not found when trying to clean variables

### ISSUE THREAD

**Issue by 875d** [IRkernel/IRkernel#726]
- [X] I followed the [official installation instructions](https://irkernel.github.io/installation/)
  
- [X] I reproduced the error with the newest versions of `IRkernel`, `IRdisplay`, and `repr`
  
- [X] I included a [minimal reproducible example](https://stackoverflow.com/a/5963610/247482)
- [X] This the right repository: I think the issue is with IRkernel, not
      [IRkernel/repr](https://github.com/IRkernel/repr/issues/new) or a third party repository
  

The following valid R code throws an error in IRkernel:

```
In [3]: rm(list = ls(all = TRUE))
Error in zmq.msg.send(parts[[i.part]], socket, flags = .pbd_env$ZMQ.SR$SNDMORE,  : 
  object '.pbd_env' not found
Calls: <Anonymous> ... send_response -> zmq.send.multipart -> zmq.msg.send
Execution halted
```

I have IRkernel 1.3.1 and I'm connecting to it with jupyter-console (same thing happens with euporie-console)

**Comment by 875d** [comment]
See also linked euporie bug report

**Comment by 875d** [comment]
This is a duplicate of #35, which was closed saying it's an upstream bug. But the bug is still here.

**Comment by flying-sheep** [comment]
Fixed in #728
